supervisor/api/supervisor.py

```diff
@@ -146,6 +146,12 @@ async def options(self, request: web.Request) -> None:
         self.sys_updater.save_data()
         self.sys_config.save_data()
 
+        # Reload the supervisor after options changed
+        await asyncio.shield(self.reload())
+
+        # Return the changed data
+        return await self.info()
```

Comment:
All `options` API work in same way. If you want that, you need update all API calls with `options` + update the API.md

Reply:
Please looking also into CLI and make sure this fronted don't show that